modules/core/shared/src/main/scala/eu/timepit/refined/string.scala

```diff
@@ -244,6 +247,17 @@ object string extends StringInference {
       )
   }
 
+  object NonEmptyFinite {
+    implicit def nonEmptyFiniteValidate[N <: Int](
```

Comment:
I'm not sure if it is possible to derive this from pre-existing `Validate` instances for `NonEmpty` and `MaxSize[N]`. I tried without it but that didn't seem to work, unless I was doing something wrong.

Reply:
Hmm yea.. seems like it was a leftover requirement from earlier experiments maybe. In any case, it's gone now :)